examples/common/python/connectors/direct/tcs_listener/tcs_work_order_handler.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2019 Intel Corporation
+# Copyright 2020 Intel Corporation
```

Comment:
This file was created in 2019 I think. So no need to change the copyright

Reply:
Makes sense. Updated.